src/DataCollection.UWP/Views/IdentifiedFeaturePopup.xaml

```diff
@@ -1,15 +1,121 @@
-﻿<UserControl
+﻿<UserControl x:Name="IdentifiedFeatureUserControl"
     x:Class="Esri.ArcGISRuntime.ExampleApps.DataCollection.UWP.Views.IdentifiedFeaturePopup"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:Esri.ArcGISRuntime.ExampleApps.DataCollection.UWP.Views"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:converters="using:Esri.ArcGISRuntime.ExampleApps.DataCollection.Shared.Converters"
     mc:Ignorable="d"
     d:DesignHeight="300"
     d:DesignWidth="400">
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <converters:DebugConverter x:Key="DebugConverter"/>
+            <converters:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter"/>
+            <converters:NullToVisibilityConverter x:Key="NullToVisibilityConverter"/>
+        </ResourceDictionary>
+    </UserControl.Resources>
 
     <Grid>
+        <Pivot>
+            <PivotItem Header="Details">
+                <ScrollViewer>
+                    <Grid>
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto"/>
+                            <RowDefinition Height="Auto"/>
+                            <RowDefinition Height="*"/>
+                        </Grid.RowDefinitions>
+                        <!--List of attributes from identified feature-->
+                        <ItemsControl ItemsSource="{x:Bind IdentifiedFeatureViewModel.PopupManager.DisplayedFields, Mode=OneWay}" Grid.Row="0">
+                            <ItemsControl.ItemTemplate>
+                                <DataTemplate>
+                                    <StackPanel>
+                                        <TextBlock Text="{Binding Field.Label, Mode=OneWay}" FontWeight="Bold" Margin="10, 0, 10, 0" />
+                                        <TextBox IsEnabled="false" Text="{Binding FormattedValue, Mode=OneWay}" TextWrapping="Wrap" BorderThickness="0" Margin="10, 0, 10, 0"/>
```

Comment:
I think setting border thickness to 0 has affected the alignment in other ways. 

![image](https://user-images.githubusercontent.com/29742178/54960178-d140b900-4f18-11e9-8cd9-b6f98261128b.png)

I tried manually setting the padding to 5 and that seemed to make it work, but if overriding border thickness can be avoided I might prefer that.

Reply:
I actually made them TextBlocks instead, I think that may work better